src/display/text_layer.js

```diff
@@ -443,9 +466,35 @@ function renderTextLayer(renderParameters) {
     viewport: renderParameters.viewport,
     textDivs: renderParameters.textDivs,
     textContentItemsStr: renderParameters.textContentItemsStr,
+    textDivProperties: renderParameters.textDivProperties,
+    isOffscreenCanvasSupported: renderParameters.isOffscreenCanvasSupported,
+    scale: renderParameters.scale,
```

Comment:
The new parameters need to be documented in the typedef: https://github.com/mozilla/pdf.js/blob/748be3f70215817ef5e4eeeec412556fef31908c/src/display/text_layer.js#L22-L42